engine/channels.go

```diff
@@ -14,13 +16,13 @@ const (
 	TestMetrics = "test-metrics"
 
 	// Channels for consensus protocols
-	ConsensusCommittee = "consensus-committee"
-	ConsensusCluster   = "consensus-cluster"
+	ConsensusCommittee     = "consensus-committee"
+	consensusClusterPrefix = "consensus-cluster" // use ChannelConsensusCluster
```

Comment:
Would be good to mention that this is a function, I was looking for it in the list of constants.

Reply:
Done in https://github.com/onflow/flow-go/pull/32/commits/2204bca415cd94cc65310414b35fa1810465efa4. Good to see you, max 😃